src/transform/class/inheritance/index.js

```diff
@@ -0,0 +1,34 @@
+import UtilInherits from './UtilInherits';
+import Prototypal from './Prototypal';
+
+/**
+ * Processes nodes to detect super classes and return information for later
+ * transformation.
+ */
+export default class Inheritance {
+  /**
+   * @param {Object} cfg
+   *   @param {PotentialClass[]} cfg.potentialClasses Class name
+   */
+  constructor({potentialClasses}) {
+    this.potentialClasses = potentialClasses;
+    this.utilInherits = new UtilInherits(this);
+    this.prototypal = new Prototypal(this);
```

Comment:
Yep, it turns out everything still works when the usage of potentialClasses is removed.
50613a8afedfd0429dda1270000bcd11c0f5a343